src/AdminRoatp/SFA.DAS.AdminRoatp.Api.UnitTests/Controllers/ProvidersAllowedListControllerTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.AdminRoatp.Api.Controllers;
+using SFA.DAS.AdminRoatp.Application.Queries.GetProvidersAllowedList;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AdminRoatp.Api.UnitTests.Controllers;
+public class ProvidersAllowedListControllerTests
+{
+    [Test, MoqAutoData]
+
+    public async Task GetAllowedList(
```

Comment:
Test  name should at least have two parts, method name and  outcome (which should be verified)